### ISSUE TITLE
mmlspark-0.12 LightGBMClassifier on Spark using scala cannot do multi-classification

### ISSUE THREAD

**Issue by puyvqi** [Azure/mmlspark#306]
``I just test iris dataset on LightGBMClassifier ,my label column has three distinct values( 1.0 , 2.0 , 0.0 )
but the output of prediction is always two values(0.0 , 1.0)(I just use the training dataset to test).
And i find the booster.numClasses 's value always is 1 ,there seem no way to specify to numclasses(like xgboost) in scala(i didn't find). 
And I cannot find any using example for scala ,all for pyspark?

```
    val indexer = new StringIndexer().setInputCol("col5").setOutputCol("label")
    val vecass = new VectorAssembler().setInputCols(Array("col1","col2","col3","col4"))
      .setOutputCol("features")
    val train1 = vecass.transform(indexer.fit(df2).transform(df2))
    val clf = new LightGBMClassifier()
      .fit(train1)
    val pred = clf.transform(train1)
    pred.show(120)
```

**Comment by imatiach-msft** [comment]
hi @puyvqi we don't have multiclass classification implemented yet, just binary and regression.  If you really needed it, you could use the OneVsAll classifier from spark ML, but it wouldn't be as fast as using lightGBM directly.  It is on our TODO list to add the multiclass classification.
For scala samples you can take a look at our tests, eg:
https://github.com/Azure/mmlspark/blob/master/src/lightgbm/src/test/scala/VerifyLightGBMClassifier.scala#L62
Hope this helps!

**Comment by puyvqi** [comment]
thanks! so fast!

**Comment by puyvqi** [comment]
I have another problem , i want to get some information of trained model (ie . importance of each feature to the result).Could you help solving this problem?
i find a method called score in com.microsoft.ml.spark.LightGBMBooster,(and think this is what i am looking for) but there is no annotation,I cannot understand its meaning : it receive a Vector and return a boolean. what is the mean of thar vector? and what does the boolean result stand for?

**Comment by mhamilton723** [comment]
@puyvqi could you make this into a separate issue please? Thank you!
